src/picongpu/include/plugins/radiation/calc_amplitude.hpp

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2013 Heiko Burau, Rene Widera, Richard Pausch
+ * Copyright 2013-2015 Heiko Burau, Rene Widera, Richard Pausch
```

Comment:
to be more correct you could write: 2013, 2015 (since no changes where made in 2014)